heapster.go

```diff
@@ -112,6 +112,7 @@ func doWork() ([]source_api.Source, sinks.ExternalSinkManager, manager.Manager,
 	// Spawn the Model Housekeeping goroutine even if the model is not enabled.
 	// This will allow the model to be activated/deactivated in runtime.
 	// Set the housekeeping period to 2 * argModelResolution + 25 sec
+	// TODO(afein): select a more well-defined housekeeping interval
```

Comment:
Restating my comment from your previous PR: If we split update to have a separate container and node level metrics storage section and aggregation section, it might be easier to deal with these durations.